src/components/CTCookieAgreement/index.js

```diff
@@ -46,87 +51,85 @@ function CTCookieAgreement() {
       fullWidth
       maxWidth='sm'
       scroll='paper'
-
+      id="cookie-agreement-inner-wrapper"
     >
-      <div id='cookie-agreement-inner-wrapper'>
```

Comment:
@aparekh5 Also, I canceled this wrapper, the `DialogActions` or `DialogContent` need to be directly placed in the `Dialog` component, so that DialogActions can be displayed sticky at the bottom (on Chrome).

Reply:
Makes sense